components/radio/demo/index.js

```diff
@@ -0,0 +1,8 @@
+import { AuroRadio } from '../src/auro-radio.js';
```

Comment:
There is a duplicate file under `./src/index.js`. I don't see the need for both. Let's pick one and keep that consistent for all the components. I would probably delete the one make that change within all components.